src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -186,7 +214,9 @@ export default function InstallModal({ appName, backdropClick }: Props) {
           </SvgButton>
           <span className="title">
             {gameInfo?.game?.title}
-            <FontAwesomeIcon icon={isMacNative ? faApple : faWindows} />
+            <FontAwesomeIcon
+              icon={isMacNative ? faApple : isLinuxNative ? faLinux : faWindows}
```

Comment:
Perhaps this could be on a function since we will also have a Heroic runner with an icon eventually :)
So something like `getIcon`